src/molecules/Footers/OppskriftFooter.js

```diff
@@ -58,15 +43,12 @@ const OppskriftLink = styled.a`
 	text-decoration: none;
 	color: inherit;
 	font-family: 'Ubuntu', sans-serif;
+	font-size: ${props => props.theme.variables.headingLargeSize};
```

Comment:
When specifying the font-size, we should probably specify a line-height as well. for a headingLargeSize, I would suggest using headingLargeLineHeight